rpcs3/Emu/System.cpp

```diff
@@ -854,4 +1007,4 @@ s32 error_code::error_report(const fmt_type_info* sup, u64 arg, const fmt_type_i
 	return static_cast<s32>(arg);
 }
 
-Emulator Emu;
+Emulator Emu;
```

Comment:
newline missing

Reply:
Will find that and will try and figure out what my Visual Studios obsession is with changing lines like this.